Detections/WindowsEvent/SOURGUM_IOC_WindowsEvent.yaml

```diff
@@ -0,0 +1,65 @@
+id: 066395ac-ef91-4993-8bf6-25c61ab0ca5a
+name: SOURGUM Actor IOC - July 2021
+description: | 
+  'Identifies a match across IOC's related to an actor tracked by Microsoft as SOURGUM'
+severity: High 
+requiredDataConnectors: 
+  - connectorId: WindowsSecurityEvents
+    dataTypes:
+      - SecurityEvents      
+queryFrequency: 6h 
+queryPeriod: 6h 
+triggerOperator: gt 
+triggerThreshold: 0 
+tactics: 
+  - Persistence
+relevantTechniques:
+  - T1546
+tags:
+  - SOURGUM
+query:  |
+  let iocs = externaldata(DateAdded:string,IoC:string,Type:string,TLP:string) [@"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Sample%20Data/Feeds/SOURGUM.csv"] with (format="csv", ignoreFirstRecord=True);
+  let domains = (iocs | where Type =~ "domainname"| project IoC);
+  let sha256Hashes = (iocs | where Type =~ "sha256" | project IoC);
+  let file_path1 = (iocs | where Type =~ "filepath1" | project IoC);
+  let file_path2 = (iocs | where Type =~ "filepath2" | project IoC);
+  let file_path3 = (iocs | where Type =~ "filepath3" | project IoC);
+  let reg_key = (iocs | where Type =~ "regkey" | project IoC);
+  WindowsEvent
+  | where EventID == 4688 and (EventData has_any (file_path1) or EventData has_any (file_path2) or  EventData has_any (file_path3) or EventData has_any ('reg add') or EventData has_any (reg_key) )
+  | extend CommandLine = tostring(EventData.CommandLine)
+  | extend NewProcessName = tostring(EventData.NewProcessName)
+  | extend ParentProcessName = tostring(EventData.ParentProcessName)
+  | where ( CommandLine has_any (file_path1)) or ( CommandLine has_any (file_path3)) or ( CommandLine has 'reg add' and CommandLine has_any (reg_key) and CommandLine has_any (file_path2)) or (NewProcessName has_any (file_path1)) or (NewProcessName has_any (file_path3)) or (ParentProcessName has_any (file_path1)) or (ParentProcessName has_any (file_path3))
```

Comment:
Style-wise, I would break this to multiple lines. It is very hard to review otherwise.